pkg/operator/ceph/cluster/cluster_external.go

```diff
@@ -145,6 +145,17 @@ func (c *ClusterController) configureExternalCephCluster(cluster *cluster) error
 			return errors.Wrap(err, "failed to create crash collector kubernetes secret")
 		}
 	}
+	// Create exporter secret
+	if !cluster.Spec.Monitoring.MetricsDisabled {
+		if cluster.ClusterInfo.CephCred.Username == client.AdminUsername &&
```

Comment:
can you please show the results in the case of non admin  user privileges, lemme know if you face any difficulty with testing

Reply:
Can I test it by update the existed secret ?

Reply:
Updating the existence might not fully work, can you try it in a sample test cluster

Reply:
@travisn https://github.com/rook/rook/pull/14456#discussion_r1721358355 I don't actually remember but the check seems tricky..

But if you see it as a low risk we can go ahead with it, and will do the follow up testing if needed

Reply:
Yes, seems low risk thanks. The CI would already be testing the non-admin case too, right?

Reply:
For canary ci it should not test, not sure if multi deploy suite test it out for non admin case

Reply:
Can we keep this change just for latest release and not backport and if we see it's stable we can backport it later?